src/util/token.js

```diff
@@ -32,9 +32,9 @@ const publicKey = (seed) => {
   ))
 }
 
-const prefix = ({ secretKey, peerPublicKey, currency }) => {
+const prefix = ({ secretKey, peerPublicKey, currencyScale, currencyCode }) => {
   const tokenPart = token(secretKey, peerPublicKey).substring(0, 5)
-  return ('peer.' + tokenPart + '.' + currency.toLowerCase() + '.')
+  return ('peer.' + tokenPart + '.' + currencyScale  + '.' + currencyCode.toLowerCase() + '.')
```

Comment:
I think the scale should go after the currency so that it's in increasing levels of specificity, i.e. peer > shared secret between these two peers > currency on this trustline > scale of currency of this trustline